src/Sarif.Viewer.VisualStudio.Core/Models/SarifErrorListItem.cs

```diff
@@ -94,8 +94,8 @@ public SarifErrorListItem(Run run, int runIndex, Result result, string logFilePa
             this.WorkingDirectory = Path.Combine(Path.GetTempPath(), this.RunIndex.ToString());
             this.HelpLink = this.Rule?.HelpUri;
 
-            this.RawMessage = result.GetMessageText(rule);
-            this.ShortMessage = this.RawMessage;
+            this.RawMessage = result.GetMessageText(rule, concise: false).Trim();
```

Comment:
>.Trim() [](http://example.com/codeflow?start=73&length=7)

to whom has concern no null check before Trim, I have checked the GetMessageText() will never return null and return string.Empty instead